### ISSUE TITLE
What is the plan for CWA?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#866]
## Your Question

Politics made clear this week, that if nothing unexpected happens, most Corona restrictions will be lifted time by time.
With this political change, the question what happens with CWA arises?
Is the project team planning to add new features to the app which supports users who want to stay careful, even after 19.03.2022? 
Or are you planning to continuing development as nothing happened? 

Some questions users ask themselves: 

- What use has the exposure logging for me as a user after 19.03.2022? - Will there be any consequences of a red warning?
- Should I still maintain the contact journal? 
- Is the check-in feature useful? CWA Check-ins are currently rare and will probably become even rarer when restrictions are lifted on 19.03.2022
- I don't need to show my EU DCCs in every day life situation, but OK, I can still have them laying around in CWA, but for what?

To summarize this issue: I want to make sure the team has a plan and does not continue development as nothing ever happened.

**Comment by Jo-Achim** [comment]
Clairvoyance is difficult. But even if it currently seems as if Corona is over... we should (better) be prepared if something similar happens to us in autumn / winter as in the last 2 years. It will probably be right to develop at least certain items on the to-do list further or to complete them and to eliminate known errors / problems.

Irrespective of this, the individual federal states should be given the opportunity via "hotspot regulations" to be able to reintroduce (previous) restrictions at any time if the corresponding incidences occur, critical infrastructures are overloaded, etc. Then the functions of the CWA would actually be in demand again or would still be in demand and important. 

This would, for example, also affect vaccination statuses and their evidence; currently older people are already being offered the 4th vaccination - i.e. the 2nd booster.
Irrespective of this, the validity of the vaccination certificates is still limited in time, which could become relevant against the background of 'new virus variants'.

**Comment by Jo-Achim** [comment]
> Anything that makes this easier and more comfortable would be most helpfull.

Right. E.g.: [Check-In / -Scan: Suggestion for functional expansion and flexibility ...](https://github.com/corona-warn-app/cwa-wishlist/issues/713) or similar.

**Comment by MikeMcC399** [comment]
https://www.n-tv.de/politik/Lauterbach-Freedom-Day-wird-es-nicht-geben-article23196500.html
"Gesundheitsminister Karl Lauterbach pocht bei den künftigen Schutzmaßnahmen weiter auf die Zuständigkeit der Bundesländer. Sie sollen die Möglichkeit nutzen, geltende Maßnahmen bis zum 2. April zu verlängern. Einen "Freedom Day" werde es somit nicht geben."

**Comment by Ein-Tim** [comment]
@MikeMcC399

Thank you.

I will close this issue now, more concrete issues should be opened for the remaining points.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
- It would be useful in any case to know what the plans are for CWA over the next weeks and months. This is a long-standing open issue. See for instance https://github.com/corona-warn-app/cwa-wishlist/issues/258

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Yes agreed. What do you think, should I leave this issue open?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> What do you think, should I leave this issue open?

If you remove "after "Freedom" day" from the title, you could leave the issue open, as the questions you ask are still relevant.

**Comment by Ein-Tim** [comment]
The federal minister of health today announced that, starting on 01.05.2022, isolating when tested positive on SARS-CoV-2 is not mandatory anymore.
After this date, I am really struggling to see the use of the CWA.

While it can certainly warn you if you met someone before they tested positive, it can't warn you if you met someone after they tested positive.

**Comment by Jo-Achim** [comment]
That's kind of right.
But if we don't want to repeat the mistakes of the past, we should at least be prepared for the coming fall. (Apart from new virus variants.)
Seen in this way, the statement seems premature to me.
> After this date, I am really struggling to see the use of the CWA.

And, as I said before, also against the background of an 'internal tidying up' within the CWA and the long wish lists, some of which should perhaps still be realised. Admittedly, this only makes sense if you want to exercise a certain amount of caution. Because the topic also includes the recognition of a new influence, the epidemiological seriousness of which cannot necessarily be assumed.

**Comment by Ein-Tim** [comment]
@Jo-Achim 

> And, as I said before, also against the background of an 'internal tidying up' within the CWA and the long wish lists, some of which should perhaps still be realised.

I definitely agree with you.
There should be a two-steps process here:
1. Do some housekeeping in the GitHub repos, fix reported bugs or close these which won't be tackled.
While this is done by the development team, RKI prepares for Phase 2;
2. Implement as many, senseful, extensions of the CWA, either proposed via the wishlist or on other ways. Maybe start a survey, which features users are missing in the CWA.

**Comment by Jo-Achim** [comment]
> The federal minister of health today announced that, starting on 01.05.2022, isolating when tested positive on SARS-CoV-2 is not mandatory anymore. After this date, I am really struggling to see the use of the CWA.

And with today's announcement, the Minister of Health - as announced yesterday (April 05, 2022) in the Markus Lanz (ZDF) program - withdrew this point again. See: [Doch keine freiwillige Corona-Isolation](https://www.tagesschau.de/inland/lauterbach-kehrtwende-isolation-coronavirus-101.html).
Isolation if the test is positive should continue to be mandatory. Only the quarantine should be 'voluntary'. The reasoning was: If the mandatory isolation in the case of a positive corona test were dropped, this would send the wrong signal in the direction of the 'harmlessness of the corona virus'.

**Comment by Ein-Tim** [comment]
This is a very unspecific issue, therefore I'm closing it now.
